### PR TITLE
hapi@7.5.3

### PR DESCRIPTION
**Pull request by robertkowalski**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ var Hapi = require('hapi'),
     config = require('./config.js'),
     getViewPath = config.getViewPath;
 
-var server = Hapi.createServer(config.host, config.port, config.server);
+var server = Hapi.createServer(config.host, config.port);
```
Comment (robin-drexler) [inline server.js:5]: Hm, we now only provide the views param from our server config to the hapi server. Aren't keys like `layoutPath` completely unused now? (I couldn't find any usage) And is that ok?

Comment (robertkowalski) [inline server.js:5]: `layoutPath` is a property from `config.server.views` which we still use (see line 6). Hapi 7 just doesn't allow to pass them on creation, so we have to call:

``` javascript
server.views(config.server.views);
```

in line 6

(see also `Server Views Configuration` in https://github.com/hapijs/hapi/issues/2023)

Comment (robin-drexler) [inline server.js:5]: Ah, I see. Didn't notice that layoutPath is underneath views.

**Comment by robin-drexler** [comment]
lgtm :)

**Comment by robertkowalski** [comment]
Yay! Thank you!

**Pull request merged**